lib/puppet/pops/issues.rb

```diff
@@ -917,5 +917,10 @@ def self.hard_issue(issue_code, *args, &block)
   LOADER_FAILURE = issue :LOADER_FAILURE, :type do
     _('Failed to load: %{type_name}') % { type: type }
   end
+
+  DEPRECATED_APP_ORCHESTRATION = issue :DEPRECATED_APP_ORCHESTRATION, :klass do
+    _("Use of the application-orchestration %{expr} is deprecated. See http://links.puppet.com/tbd-app-orchestration-deprecation" % { expr: label(klass) })
```

Comment:
@jbondpdx could you create a link for the orchestration deprecation?

Reply:
Deprecations are in the release notes, so the URL would be version specific, like https://puppet.com/docs/puppet/6.1/release_notes_puppet.html#concept-3883. Note that this assumes I get release note information in the related tickets and any applicable predocs. What version is the deprecation effective in?

Reply:
We've used `links.puppet.com` before for some deprecations. Is that what you were wanting @hlindberg?

```
lib/puppet/network/http/request.rb:    # YAML in network requests is not supported. See http://links.puppet.com/deprecate_yaml_on_network
lib/puppet/parser/ast/node.rb:      raise Puppet::DevError, _("Node inheritance is removed in Puppet 4.0.0. See http://links.puppet.com/puppet-node-inheritance-deprecation")
lib/puppet/pops/issues.rb:    _("The operator '%{operator}' is no longer supported. See http://links.puppet.com/remove-plus-equals") % { operator: operator }
lib/puppet/pops/issues.rb:    _("Use of 'import' has been discontinued in favor of a manifest directory. See http://links.puppet.com/puppet-import-deprecation")
lib/puppet/pops/issues.rb:    _("Node inheritance is not supported in Puppet >= 4.0.0. See http://links.puppet.com/puppet-node-inheritance-deprecation")
```

Reply:
Hm, isn't "links.puppet.com" the old way? I remember that we recently did a similar thing - it was a link to a page in documentation for that deprecation. I don't really care what it points to as long it is stable over (long) time.

Reply:
Ah yeah, not sure what current practice is. But agree about needing a stable link.